lib/features/children/details/widgets/child_cancel_rga_failed_dialog.dart

```diff
@@ -15,7 +15,10 @@ class ChildCancelRGAFailedDialog extends StatelessWidget {
         child: Column(
           mainAxisSize: MainAxisSize.min,
           children: [
-            warningIcon(),
+            warningIcon(
```

Comment:
Are you sure the icon should scale? In figma it looks fixed size :)

Reply:
We had a call with And y and decided to try this because with fixed size as it was it looked small on some phones